src/main/java/george/backend/hiring/JavaApplication.java

```diff
@@ -0,0 +1,9 @@
+package george.backend.hiring;
```

Comment:
ℹ️ Before, the classes were in a default package (antipattern, as it messes up with visibility modifiers and would cause naming conflicts if published as a library). 

Many candidates did not move it to their own package.

Reply:
This was intentional in order to see if candidates are fixing this.

Reply:
Ok. I will remove it.